### PR TITLE
Generate provider map

### PR DESCRIPTION
**Pull request by fabianvf**
Generates a provider map on the scrAPI side, so that this logic can be removed from the OSF. Includes favicons. Relies on registry PR.

### CODE REVIEW

File: scrapi/harvesters/arxiv.py [hunk 1]
```diff
@@ -11,8 +11,9 @@
 
 
 class ArxivHarvester(OAIHarvester):
-    short_name = 'arxiv_oai'
-    long_name = 'arXiv'
+    short_name = 'arxiv'
```
Comment (chrisseto) [inline scrapi/harvesters/arxiv.py]: This will require a migration

File: scrapi/harvesters/cmu.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 class CMUHarvester(OAIHarvester):
     short_name = 'cmu'
     long_name = 'Carnegie Mellon University'
+    url = 'Carnegie Mellon University Research Showcase'
```
Comment (chrisseto) [inline scrapi/harvesters/cmu.py]: This isn't a url

File: scrapi/settings/travis-dist.py [hunk 3]
```diff
@@ -13,6 +13,8 @@
 
 USE_FLUENTD = False
 
+PLOS_API_KEY = None
```
Comment (chrisseto) [inline scrapi/settings/travis-dist.py]: No longer relevant.

File: scrapi/harvesters/arxiv.py (hunk 1 above)
Comment (fabianvf) [inline scrapi/harvesters/arxiv.py]: right, good point

**Pull request merged**